session_linux.go

```diff
@@ -60,15 +60,17 @@ func newPASession(
 	return s
 }
 
-func newMasterSession(
+func newPaDevice(
```

Comment:
`newPADeviceSession` (abbreviations in name should be [all uppercase](https://github.com/golang/go/wiki/CodeReviewComments#initialisms)).